companies/bigpoint.json

```diff
@@ -0,0 +1,40 @@
+{
+    "slug": "bigpoint",
+    "relevant-countries": [
+        "bg",
+        "cz",
+        "dk",
+        "de",
+        "gr",
+        "gb",
+        "es",
+        "fi",
+        "fr",
+        "hu",
+        "it",
+        "nl",
+        "no",
+        "pl",
+        "pt",
+        "ro",
+        "sk",
+        "se"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Bigpoint GmbH",
+    "address": "Drehbahn 47-48\n20354 Hamburg\nDeutschland",
+    "phone": "+49 40 8814130",
+    "fax": "+49 40 881413 906",
+    "email": "privacy@bigpoint.net",
+    "web": "https://www.bigpoint.net/",
+    "sources": [
+        "https://legal.bigpoint.com/DE/privacy-policy/de-DE",
+        "https://legal.bigpoint.com/DE/terms-and-conditions/de-DE",
+        "https://legal.bigpoint.com/DE/imprint/de-DE"
```

Comment:
From what I've seen, the latter two sources don't have any new information that isn't already covered in the privacy policy. If so, you don't need to add them.

Reply:
removed